src/CircuitBreaker/src/HystrixBase/Steeltoe.CircuitBreaker.HystrixBase.csproj

```diff
@@ -1,26 +1,26 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\..\..\versions.props" />
   <PropertyGroup>
-    <Description>Steeltoe Neflix Hystrix Client - Base Package</Description>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
-    <AssemblyName>Steeltoe.CircuitBreaker.HystrixBase</AssemblyName>
-    <PackageId>Steeltoe.CircuitBreaker.HystrixBase</PackageId>
+    <RootNamespace>Steeltoe.CircuitBreaker.Hystrix</RootNamespace>
+    <Description>An implementation of Neflix's Hystrix, for .NET</Description>
     <PackageTags>Spring Cloud;Netflix;Hystrix Client;Circuit Breaker</PackageTags>
```

Comment:
Add aspnetcore?

Reply:
The base package doesn't have anything specific to asp.net core